app/assets/javascripts/admins/events.js

```diff
@@ -0,0 +1,22 @@
+$(document).on('turbolinks:load', function(){
```

Comment:
1. Use ECMAScript 6  syntax
2. Create split this in config/color_picker.js and config/dateTimePicker.js